rho/profileclearcommand.py

```diff
@@ -61,75 +82,43 @@ def _validate_options(self):
             self.parser.print_help()
             sys.exit(1)
 
-    # pylint: disable=too-many-branches,too-many-locals
     def _do_command(self):
-        profiles_list = []
+        if not os.path.isfile(utilities.PROFILES_PATH):
```

Comment:
Printing this message is not dependent on the options when there PROFILE_PATH does not exist.